README.md

```diff
@@ -51,49 +51,63 @@ env_var.sh
 #### Run the translator script (from Work_Zone)
 
 ```
-python -m translator.source_code.icone_translator -i inputfile.xml -o outputfile.geojson
+python -m translator.icone_translator inputfile.xml --outputFile outputfile.geojson
 ```
 
 Example usage:
 
 ```
-python -m translator.source_code.icone_translator -i 'translator/sample files/icone data/incidents_extended.xml' 
+python -m translator.icone_translator 'translator/sample files/icone data/incidents_extended.xml' 
```

Comment:
On this example usage and the ones below for cotrip and navjoy, are we missing the output file name? Or is there some default output value used so we don't have to provide?

Reply:
There is a default outputFile for each of the scripts